### ISSUE TITLE
VSCode locks up with lots of console.log / expect failures

### ISSUE THREAD

**Issue by danzel** [wallabyjs/public#500]
### Issue description or question

Have this test:
```ts
describe("console.log logs", () => {
	it('locks up visual studio code', () => {
		for (let i = 0; i < 100000; i++) {
			console.log('hi');
			expect(true).toBe(false);
		}
	});
});
```
Cut the expect line and move it to after the loop.
VS Code locks up. (May happen slightly before or after moving the line around).

I've ran in to this a few times when accidentally adding logging in to an area that gets ran lots of times.

### Wallaby.js configuration file

```javascript
module.exports = function (w) {
	return {
		files: [
			'app/**/*.ts',
			'test/util/*.ts'
		],
		tests: [
			'test/**/*Spec.ts'
		],
		testFramework: 'jasmine',
		// TypeScript compiler is on by default with default options,
		// you can configure built-in compiler by passing options to it
		// See interface CompilerOptions in
		// https://github.com/Microsoft/TypeScript/blob/master/src/compiler/types.ts
		compilers: {
		  '**/*.ts': w.compilers.typeScript({
			  module: 1 //commonjs
		  })
		},
		env: {
			type: 'node'
		}
	};
};
```

### Code editor or IDE name and version

Visual Studio Code v0.10.10

### OS name and version

Windows 8.1

(Thanks for this awesome tool)

**Comment by ArtemGovorov** [comment]
The issue is fixed in the latest core v1.0.196 (will updated automatically).

**Comment by danzel** [comment]
Awesome thanks @ArtemGovorov
